tests/mocks/types_handler.go

```diff
@@ -37,9 +37,10 @@ func (m *MockAnteDecorator) EXPECT() *MockAnteDecoratorMockRecorder {
 // AnteHandle mocks base method.
 func (m *MockAnteDecorator) AnteHandle(ctx types.Context, tx types.Tx, simulate bool, next types.AnteHandler) (types.Context, error) {
 	m.ctrl.T.Helper()
-	m.ctrl.Call(m, "AnteHandle", ctx, tx, simulate, next)
-	// NOTE: we need to edit a generated code to call the "next handler"
-	return next(ctx, tx, simulate)
+	ret := m.ctrl.Call(m, "AnteHandle", ctx, tx, simulate, next)
+	ret0, _ := ret[0].(types.Context)
+	ret1, _ := ret[1].(error)
+	return ret0, ret1
```

Comment:
I expect it to behave differently than before, what is the intent of this fix?

Reply:
I'm surprised you noticed. It appears that this code generated by the mock was manually modified in Tendermint. It may need to be fixed.